### DISCUSSION TITLE
Baram Mesh and Flow, adding shortcuts to common tasks (rename, edit, delete)

### DISCUSSION THREAD

**Discussion by SergioP1975** [nextfoam/baram#143] [Ideas]
Currently for renaming objects (patches, bc....) there is no shortcut assigned, so maybe could be implemented the standard F2 (at least for Windows) for renaming, and maybe F3 for editing values of the object. For deleting...just pressing the Del key and with a confirmation message could help also.

Best regards, and thanks in advance.

**Comment by kdg5424** [comment]
Hello @SergioP1975. This is Donggyu from NEXTFOAM.
Thank you for your suggestion.
I'm not sure whether the boundary name can be edited, so I will discuss with our developers.

If it's possible, we'll develop it.

Best regards

Donggyu Kim

**Reply by SergioP1975** [reply]
Thanks!
